editor/editor_data.h

```diff
@@ -122,8 +124,27 @@ class EditorData {
 		uint64_t last_checked_version = 0;
 	};
 
+	enum ContextMenuSlot {
+		CONTEXT_SLOT_SCENE_TREE,
+		CONTEXT_SLOT_FILESYSTEM,
+		CONTEXT_SLOT_SCRIPT_EDITOR,
+		CONTEXT_SUBMENU_SLOT_FILESYSTEM_CREATE,
+	};
+
+	enum {
+		CONTEXT_MENU_ITEM_ID_BASE = 200
```

Comment:
I'd make it 1000, just to be safe.
This can also be `inline static constexpr int`.